### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by Vadoomz**
I've done smth wrong with PR but please check only HW_5 file

### CODE REVIEW

File: HW_5.py [hunk 1]
```diff
@@ -3,6 +3,31 @@
 # це “буква” + яка вона (велика чи маленька),
 # це “символ”
 #
+def symbol_checker():
+    enter = input("Enter your text ")
+    count = len(enter)
+    for x in enter:
+        if (x.isdigit() == True):
```
Comment (klymenok) [inline HW_5.py]: There is no need to use parentheses here. Also, You can skip `== True` since `if` does this check in the end

File: HW_5.py [hunk 2]
```diff
@@ -3,6 +3,31 @@
...
+            if (int(x) % 2 == 0):
+                print(f'{x} is even')
+            elif (int(x) % 2 != 0):
+                print(f'{x} is odd')
+        elif (x.isalpha() == True):
```
Comment (klymenok) [inline HW_5.py]: There is no need to use parentheses here. Also, You can skip == True since if does this check in the end

File: HW_5.py [hunk 3]
```diff
@@ -3,6 +3,31 @@
...
+                if x.isupper() == True:
+                    print(f'{x} Bigdaddy letter')
+                elif (x.islower() == True):
```
Comment (klymenok) [inline HW_5.py]: There is no need to use parentheses here. Also, You can skip == True since if does this check in the end

File: HW_5.py [hunk 4]
```diff
@@ -3,6 +3,31 @@
...
+                    print(f'{x} small letter')
+        else:
+            print(f'{x} symbol')
+symbol_checker()
+
 # 2. Створити програму, яка буде друкувати в консоль “I love Python” кожні 4.2 секунди, поки її виконання не буде перервано вручну.
 # Підказка: для того, щоб програма могла зупинитися на вказаний час, можна використати бібліотеку time (import time), а саме функцію sleep().
 # (time.sleep(second), де second, це кількість секунд, які програма має чекати).
+
+# import time as time
```
Comment (klymenok) [inline HW_5.py]: Why this code is commented out?

**Review by klymenok — CHANGES_REQUESTED** [review]

File: HW_5.py (hunk 4 above)
Comment (Vadoomz) [reply HW_5.py]: I tested only 1st code so commented 2d one

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**